### DISCUSSION TITLE
How to Restore Disk from Another clickhouse instance

### DISCUSSION THREAD

**Discussion by ehddnr301** [ClickHouse/ClickHouse#67951] [Q&A] [answered]
Maybe duplicate question #61283 

![image](https://github.com/user-attachments/assets/f6419fd8-378b-49e5-8633-37a35dab10c2)

The external disk I used is stored in azure blob storage in a different format than normal csv, but if my instance is down and I want to re-table with that data, I think I need to connect the metadata properly with the data stored in the blob storage, what should I do?

For additional information, I am using clickhouse as a single node.

And using the move_factor value, some data is stored in ssd and some data is stored in object storage.

If I have a problem with my vm and the clickhouse is currently unavailable and I'm wondering what I need to do to use the data stored on the object storage in another vm.

**Comment by den-crane** [comment, marked as answer]
>my instance is down 

If you lost a local disk and you don't have a backup of the local disk then the data is lost for good.

Clickhouse stores metadata locally to be able to read objects stored in azure blob storage.

**Reply by ehddnr301** [reply]
I asked the question and thought about it more.
using disk and blob storage itself didn't seem to be the purpose of backup. That's why it's tricky when I try to use disk to back up
Thanks a lot
